src/components/dialogs/StartPrintDialog.vue

```diff
@@ -119,8 +119,37 @@ export default class StartPrintDialog extends Mixins(BaseMixin) {
         return this.$t('Dialogs.StartPrint.DoYouWantToStartFilename', { filename: this.file?.filename ?? 'unknown' })
     }
 
+    get fileTimestamp() {
+        return typeof this.file.modified.getTime === 'function' ? this.file.modified.getTime() : 0
+    }
+
+    get thumbnails() {
+        return this.file.thumbnails ?? []
+    }
+
+    get bigThumbnail() {
+        return this.thumbnails.find((thumbnail) => thumbnail.width >= thumbnailBigMin)
+    }
+
+    get currentPathWithSlash() {
```

Comment:
change name to match name and function of the getter
```suggestion
    get currentPathWithoutSlash() {
```

Reply:
@meteyou